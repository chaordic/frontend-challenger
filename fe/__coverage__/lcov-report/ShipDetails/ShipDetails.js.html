<!doctype html>
<html lang="en">
<head>
    <title>Code coverage report for ShipDetails/ShipDetails.js</title>
    <meta charset="utf-8" />
    <link rel="stylesheet" href="../prettify.css" />
    <link rel="stylesheet" href="../base.css" />
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <style type='text/css'>
        .coverage-summary .sorter {
            background-image: url(../sort-arrow-sprite.png);
        }
    </style>
</head>
<body>
<div class='wrapper'>
  <div class='pad1'>
    <h1>
      <a href="../index.html">All files</a> / <a href="index.html">ShipDetails</a> ShipDetails.js
    </h1>
    <div class='clearfix'>
      <div class='fl pad1y space-right2'>
        <span class="strong">80% </span>
        <span class="quiet">Statements</span>
        <span class='fraction'>4/5</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">84% </span>
        <span class="quiet">Branches</span>
        <span class='fraction'>21/25</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">33.33% </span>
        <span class="quiet">Functions</span>
        <span class='fraction'>1/3</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">80% </span>
        <span class="quiet">Lines</span>
        <span class='fraction'>4/5</span>
      </div>
    </div>
  </div>
  <div class='status-line high'></div>
<pre><table class="coverage">
<tr><td class="line-count quiet">1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
32
33
34
35
36
37
38
39
40
41
42
43
44
45
46
47
48
49
50
51
52
53
54
55
56
57
58
59
60
61
62
63
64
65
66
67
68
69
70
71
72
73
74
75
76
77
78
79
80
81
82
83
84
85
86
87
88
89
90
91
92
93
94
95
96
97
98
99
100
101
102
103
104
105
106
107
108
109
110
111
112
113
114
115
116
117
118
119
120
121</td><td class="line-coverage quiet"><span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-yes">2x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-yes">2x</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-yes">2x</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span></td><td class="text"><pre class="prettyprint lang-js">import React, { Fragment } from 'react';
import { object, func, string } from 'prop-types';
import cssModules from 'react-css-modules';
import ShipTableItems from '../ShipTableItems/ShipTableItems';
&nbsp;
import styles from './styles.scss';
&nbsp;
const ShipDetails = ({ id, fulfillment, onToggle }) =&gt; (
  &lt;Fragment&gt;
    &lt;section className={styles.menu} id={`shipItem-${fulfillment.id}`}&gt;
      &lt;button onClick={<span class="fstat-no" title="function not covered" >e </span>=&gt; <span class="cstat-no" title="statement not covered" >onToggle(fulfillment.id, e)}</span>&gt;
        &lt;i className={`${styles.angle} ${fulfillment.open ? styles.down : <span class="branch-1 cbranch-no" title="branch not covered" >styles.up}</span>`} /&gt;
      &lt;/button&gt;
      &lt;section&gt;
        &lt;p className={`${styles.addressTitle} ${styles.green}`}&gt;
          {`Entrega ${fulfillment.id}`}
        &lt;/p&gt;
        &lt;p className={`${styles.address}`}&gt;
          {`${id}-${fulfillment.id}`}
        &lt;/p&gt;
      &lt;/section&gt;
&nbsp;
      &lt;section&gt;
        &lt;p className={styles.addressTitle}&gt;
          Status da entrega
        &lt;/p&gt;
        &lt;p className={`${styles.address} ${fulfillment.type &amp;&amp; styles[fulfillment.type.toLowerCase()]}`}&gt;
          {fulfillment.type}
        &lt;/p&gt;
      &lt;/section&gt;
    &lt;/section&gt;
&nbsp;
    &lt;section className={`${styles.main} ${fulfillment.open ? `${styles.fulfillmentOpen}` : <span class="branch-1 cbranch-no" title="branch not covered" >`${styles.fulfillmentClose}`}</span>`}&gt;
      &lt;h1&gt;Dados da entrega&lt;/h1&gt;
      &lt;article&gt;
        {fulfillment.shippedBy !== undefined &amp;&amp;
<span class="branch-1 cbranch-no" title="branch not covered" >          &lt;section&gt;</span>
            &lt;p className={styles.addressTitle}&gt;Retirado por&lt;/p&gt;
            &lt;p className={styles.address}&gt;{fulfillment.shippedBy.name}&lt;/p&gt;
            &lt;p className={styles.address}&gt;{fulfillment.shippedBy.cpf}&lt;/p&gt;
          &lt;/section&gt;
        }
&nbsp;
        {fulfillment.locationType !== undefined &amp;&amp;
          &lt;section&gt;
            &lt;p className={styles.addressTitle}&gt;Modalidade&lt;/p&gt;
            &lt;p className={styles.address}&gt;{fulfillment.locationType}&lt;/p&gt;
          &lt;/section&gt;
        }
&nbsp;
        {fulfillment.processedAt !== undefined &amp;&amp;
          &lt;section&gt;
            &lt;p className={styles.addressTitle}&gt;Data Previsão Cliente&lt;/p&gt;
            &lt;p className={styles.address}&gt;{fulfillment.processedAt}&lt;/p&gt;
          &lt;/section&gt;
        }
&nbsp;
        {fulfillment.formattedAddress !== undefined &amp;&amp;
          &lt;section&gt;
            &lt;p className={styles.addressTitle}&gt;Endereço de entrega&lt;/p&gt;
            &lt;p className={styles.address}&gt;{fulfillment.formattedAddress}&lt;/p&gt;
          &lt;/section&gt;
        }
&nbsp;
        {fulfillment.freight !== undefined &amp;&amp;
          fulfillment.freight.system !== undefined &amp;&amp;
<span class="branch-2 cbranch-no" title="branch not covered" >          &lt;section&gt;</span>
            &lt;p className={styles.addressTitle}&gt;Transportadora&lt;/p&gt;
            &lt;p className={styles.address}&gt;{fulfillment.freight.system}&lt;/p&gt;
          &lt;/section&gt;
        }
&nbsp;
        {fulfillment.type !== undefined &amp;&amp;
          &lt;section&gt;
            &lt;p className={styles.addressTitle}&gt;Tipo&lt;/p&gt;
            &lt;p className={styles.address}&gt;{fulfillment.type}&lt;/p&gt;
          &lt;/section&gt;
        }
&nbsp;
        {fulfillment.freight !== undefined &amp;&amp;
          fulfillment.freight.totalPrice !== undefined &amp;&amp;
          &lt;section&gt;
            &lt;p className={styles.addressTitle}&gt;Preço do frete&lt;/p&gt;
            &lt;p className={styles.address}&gt;{fulfillment.freight.totalPrice}&lt;/p&gt;
          &lt;/section&gt;
        }
&nbsp;
        {fulfillment.freight !== undefined &amp;&amp;
          fulfillment.freight.deliveryEstimatedDate !== undefined &amp;&amp;
          &lt;section&gt;
            &lt;p className={styles.addressTitle}&gt;Data Previsão Transportadora&lt;/p&gt;
            &lt;p className={styles.address}&gt;{fulfillment.freight.deliveryEstimatedDate}&lt;/p&gt;
          &lt;/section&gt;
        }
&nbsp;
      &lt;/article&gt;
&nbsp;
      &lt;hr /&gt;
&nbsp;
      &lt;ShipTableItems
        items={fulfillment.items}
        geral={fulfillment.geral}
      /&gt;
&nbsp;
    &lt;/section&gt;
  &lt;/Fragment&gt;
);
&nbsp;
ShipDetails.propTypes = {
  fulfillment: object.isRequired,
  onToggle: func,
  id: string,
};
&nbsp;
ShipDetails.defaultProps = {
  onToggle: <span class="fstat-no" title="function not covered" >()</span> =&gt; {},
  id: '',
};
&nbsp;
export default cssModules(ShipDetails, styles, { allowMultiple: true });
&nbsp;</pre></td></tr>
</table></pre>
<div class='push'></div><!-- for sticky footer -->
</div><!-- /wrapper -->
<div class='footer quiet pad2 space-top1 center small'>
  Code coverage
  generated by <a href="https://istanbul.js.org/" target="_blank">istanbul</a> at Mon Nov 11 2019 23:02:12 GMT+0000 (WET)
</div>
</div>
<script src="../prettify.js"></script>
<script>
window.onload = function () {
        if (typeof prettyPrint === 'function') {
            prettyPrint();
        }
};
</script>
<script src="../sorter.js"></script>
</body>
</html>
