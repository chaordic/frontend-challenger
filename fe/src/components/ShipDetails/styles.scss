@import '../../global-styles/settings.global.scss';

.menu {
    display: grid;
    grid-column-gap: 1rem;
    grid-template-columns: 1fr 3fr 3fr 3fr 3fr 3fr;
    margin: 10px 20px;
    -webkit-box-shadow: 0px 0px 10px 0px rgba(204,204,204,1);
    -moz-box-shadow: 0px 0px 10px 0px rgba(204,204,204,1);
    box-shadow: 0px 0px 10px 0px rgba(204,204,204,1);
    margin: 10px 20px 0 20px;
    grid-template-rows: 100px 1fr;
}

.fulfillmentClose {
    display: none;
    transition: all 0.3s ease; 
}

.fulfillmentOpen {
    display: block;
    transition: all 0.3s ease; 
}

.menu section {
    align-self: center;
}

.menu button {
    background-color: #f6f6f6;
    outline: none;
    cursor: pointer;
}

.main {
    -webkit-box-shadow: var(--shadow);
    -moz-box-shadow: var(--shadow);
    box-shadow: var(--shadow);
    padding: 20px;
    margin: 0 20px 10px 20px;
    transition: all 0.3s ease; 
}

.main > article {
    display: grid;
    grid-column-gap: 1rem;
    grid-template-columns: 3fr 3fr 3fr 3fr;
}

.main h1 {
    padding: 0 0 10px 0;
    font-size: 18px;
    font-weight: 500;
}
