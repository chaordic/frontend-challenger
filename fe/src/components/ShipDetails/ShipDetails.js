import React, { Fragment } from 'react';
import { object, func, string } from 'prop-types';
import cssModules from 'react-css-modules';
import ShipTableItems from '../ShipTableItems/ShipTableItems';

import styles from './styles.scss';

const ShipDetails = ({ id, fulfillment, onToggle }) => (
  <Fragment>
    <section className={styles.menu} id={`shipItem-${fulfillment.id}`}>
      <button onClick={e => onToggle(fulfillment.id, e)}>
        <i className={`${styles.angle} ${fulfillment.open ? styles.down : styles.up}`} />
      </button>
      <section>
        <p className={`${styles.addressTitle} ${styles.green}`}>
          {`Entrega ${fulfillment.id}`}
        </p>
        <p className={`${styles.address}`}>
          {`${id}-${fulfillment.id}`}
        </p>
      </section>

      <section>
        <p className={styles.addressTitle}>
          Status da entrega
        </p>
        <p className={`${styles.address} ${fulfillment.type && styles[fulfillment.type.toLowerCase()]}`}>
          {fulfillment.type}
        </p>
      </section>
    </section>

    <section className={`${styles.main} ${fulfillment.open ? `${styles.fulfillmentOpen}` : `${styles.fulfillmentClose}`}`}>
      <h1>Dados da entrega</h1>
      <article>
        {fulfillment.shippedBy !== undefined &&
          <section>
            <p className={styles.addressTitle}>Retirado por</p>
            <p className={styles.address}>{fulfillment.shippedBy.name}</p>
            <p className={styles.address}>{fulfillment.shippedBy.cpf}</p>
          </section>
        }

        {fulfillment.locationType !== undefined &&
          <section>
            <p className={styles.addressTitle}>Modalidade</p>
            <p className={styles.address}>{fulfillment.locationType}</p>
          </section>
        }

        {fulfillment.processedAt !== undefined &&
          <section>
            <p className={styles.addressTitle}>Data Previsão Cliente</p>
            <p className={styles.address}>{fulfillment.processedAt}</p>
          </section>
        }

        {fulfillment.formattedAddress !== undefined &&
          <section>
            <p className={styles.addressTitle}>Endereço de entrega</p>
            <p className={styles.address}>{fulfillment.formattedAddress}</p>
          </section>
        }

        {fulfillment.freight !== undefined &&
          fulfillment.freight.system !== undefined &&
          <section>
            <p className={styles.addressTitle}>Transportadora</p>
            <p className={styles.address}>{fulfillment.freight.system}</p>
          </section>
        }

        {fulfillment.type !== undefined &&
          <section>
            <p className={styles.addressTitle}>Tipo</p>
            <p className={styles.address}>{fulfillment.type}</p>
          </section>
        }

        {fulfillment.freight !== undefined &&
          fulfillment.freight.totalPrice !== undefined &&
          <section>
            <p className={styles.addressTitle}>Preço do frete</p>
            <p className={styles.address}>{fulfillment.freight.totalPrice}</p>
          </section>
        }

        {fulfillment.freight !== undefined &&
          fulfillment.freight.deliveryEstimatedDate !== undefined &&
          <section>
            <p className={styles.addressTitle}>Data Previsão Transportadora</p>
            <p className={styles.address}>{fulfillment.freight.deliveryEstimatedDate}</p>
          </section>
        }

      </article>

      <hr />

      <ShipTableItems
        items={fulfillment.items}
        geral={fulfillment.geral}
      />

    </section>
  </Fragment>
);

ShipDetails.propTypes = {
  fulfillment: object.isRequired,
  onToggle: func,
  id: string,
};

ShipDetails.defaultProps = {
  onToggle: () => {},
  id: '',
};

export default cssModules(ShipDetails, styles, { allowMultiple: true });
