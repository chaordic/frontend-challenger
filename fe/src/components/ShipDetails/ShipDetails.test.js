import React from 'react';
import renderer from 'react-test-renderer';
import ShipDetails from './ShipDetails';

const props = {
  onToggle: jest.fn(),
  id: '245',
  fulfillment: {
    id: 'F1',
    locationId: '359',
    shipment: {
      zip: '00000-000',
      city: 'São Paulo',
      state: 'SP',
      number: '333',
      status: 'ACTIVE',
      country: 'Brasil',
      address1: 'Rua Oscar Freire',
      name: 'Renato Pereira',
      telephone: {
        type: 'billing',
        number: '(11) 98376-6343',
      },
    },
    status: 'ENTREGUE',
    type: 'ENVIADO',
    createdAt: '02/02/2019',
    updatedAt: '02/02/2019',
    freightCosts: {
      totalTime: 11,
      totalPrice: 10,
      deliveryEstimatedDate: '2019-03-05T19:30:00.000Z',
    },
    processedAt: '02/03/2019',
    locationType: 'Own Store',
    items: [{
      sku: 'AR384675',
      quantity: 2,
      stockType: 'PHYSICAL',
      name: 'Tenis Coca Coca Loretto - Feminino',
      size: '39',
      color: 'Branco, Cinza',
      image: 'http://image.url',
      price: 'R$100,00',
      desc: 'Branco, Cinza, 39',
    }, {
      sku: 'AR384677',
      quantity: 1,
      stockType: 'PHYSICAL',
      name: 'Tênis New Balance ML501 - Masculino',
      size: '40',
      color: 'Preto',
      image: 'http://image.url',
      price: 'R$100,00',
      desc: 'Preto, 40',
    }],
    geral: {
      quantity: 3,
      items: 2,
      totalFreight: 'R$0,00',
      totalPrice: 'R$200,00',
      total: 'R$200,00',
    },
    open: true,
    formattedAddress: 'Rua Oscar Freire, 333 São Paulo - SP - 00000-000 - Brasil',
    freight: {
      deliveryEstimatedDate: '02/03/2019',
      totalPrice: 'R$10,00',
    },
  },
};

describe('(Component) ShipDetails', () => {
  it('should render an ShipDetails with result', () => {
    const tree = renderer.create(<ShipDetails {...props} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
