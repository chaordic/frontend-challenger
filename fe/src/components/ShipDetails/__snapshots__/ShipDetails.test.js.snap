// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`(Component) ShipDetails should render an ShipDetails with result 1`] = `
Array [
  <section
    id="shipItem-F1"
  >
    <button
      onClick={[Function]}
    >
      <i
        className="undefined undefined"
      />
    </button>
    <section>
      <p
        className="undefined undefined"
      >
        Entrega F1
      </p>
      <p
        className="undefined"
      >
        245-F1
      </p>
    </section>
    <section>
      <p>
        Status da entrega
      </p>
      <p
        className="undefined undefined"
      >
        ENVIADO
      </p>
    </section>
  </section>,
  <section
    className="undefined undefined"
  >
    <h1>
      Dados da entrega
    </h1>
    <article>
      <section>
        <p>
          Modalidade
        </p>
        <p>
          Own Store
        </p>
      </section>
      <section>
        <p>
          Data Previsão Cliente
        </p>
        <p>
          02/03/2019
        </p>
      </section>
      <section>
        <p>
          Endereço de entrega
        </p>
        <p>
          Rua Oscar Freire, 333 São Paulo - SP - 00000-000 - Brasil
        </p>
      </section>
      <section>
        <p>
          Tipo
        </p>
        <p>
          ENVIADO
        </p>
      </section>
      <section>
        <p>
          Preço do frete
        </p>
        <p>
          R$10,00
        </p>
      </section>
      <section>
        <p>
          Data Previsão Transportadora
        </p>
        <p>
          02/03/2019
        </p>
      </section>
    </article>
    <hr />
    <h1>
      Detalhes da entrega
    </h1>
    <table>
      <thead
        className="undefined"
      >
        <tr>
          <td>
            PRODUTO
          </td>
          <td>
            SKU
          </td>
          <td>
            QTD.
          </td>
          <td>
            PREÇO
          </td>
        </tr>
      </thead>
      <tbody>
        <tr
          className="undefined "
        >
          <td>
            <img
              alt="Tenis Coca Coca Loretto - Feminino"
              src="http://image.url"
              title="Tenis Coca Coca Loretto - Feminino"
            />
            <section>
              <p
                className="undefined"
              >
                Tenis Coca Coca Loretto - Feminino
              </p>
              <p
                className="undefined"
              >
                Branco, Cinza, 39
              </p>
            </section>
          </td>
          <td>
            <p
              className="undefined"
            >
              AR384675
            </p>
          </td>
          <td>
            <p
              className="undefined"
            >
              2
            </p>
          </td>
          <td>
            <section>
              <p>
                Subtotal:
              </p>
              <p
                className="undefined undefined"
              >
                R$100,00
              </p>
              <p>
                Frete:
              </p>
              <p
                className="undefined undefined"
              >
                R$100,00
              </p>
              <p>
                Valor total:
              </p>
              <p
                className="undefined undefined"
              >
                R$100,00
              </p>
            </section>
          </td>
        </tr>
        <tr
          className="undefined undefined"
        >
          <td>
            <img
              alt="Tênis New Balance ML501 - Masculino"
              src="http://image.url"
              title="Tênis New Balance ML501 - Masculino"
            />
            <section>
              <p
                className="undefined"
              >
                Tênis New Balance ML501 - Masculino
              </p>
              <p
                className="undefined"
              >
                Preto, 40
              </p>
            </section>
          </td>
          <td>
            <p
              className="undefined"
            >
              AR384677
            </p>
          </td>
          <td>
            <p
              className="undefined"
            >
              1
            </p>
          </td>
          <td>
            <section>
              <p>
                Subtotal:
              </p>
              <p
                className="undefined undefined"
              >
                R$100,00
              </p>
              <p>
                Frete:
              </p>
              <p
                className="undefined undefined"
              >
                R$100,00
              </p>
              <p>
                Valor total:
              </p>
              <p
                className="undefined undefined"
              >
                R$100,00
              </p>
            </section>
          </td>
        </tr>
      </tbody>
      <tfoot>
        <tr>
          <td />
          <td />
          <td />
          <td>
            <p
              className="undefined undefined"
            >
              3
               unidades de 
              2
               items
            </p>
            <section>
              <p>
                Subtotal:
              </p>
              <p
                className="undefined undefined"
              >
                R$200,00
              </p>
              <p>
                Frete total:
              </p>
              <p
                className="undefined undefined"
              >
                R$0,00
              </p>
              <p>
                Valor total:
              </p>
              <p
                className="undefined undefined"
              >
                R$200,00
              </p>
            </section>
          </td>
        </tr>
      </tfoot>
    </table>
  </section>,
]
`;
